uptycs/models.go

```diff
@@ -298,6 +298,28 @@ type Roles struct {
 	Limit  int        `json:"limit,omitempty"`
 }
 
+type ComplianceProfile struct {
+	ID                            string        `json:"id"`
+	Name                          string        `json:"name,omitempty"`
+	Description                   string        `json:"description,omitempty"`
+	Custom                        bool          `json:"custom"`
+	Priority                      int           `json:"priority,omitempty"`
+	CreatedBy                     string        `json:"createdBy"`
+	UpdatedBy                     string        `json:"updatedBy"`
+	CreatedAt                     string        `json:"createdAt"`
+	UpdatedAt                     string        `json:"updatedAt"`
+	Links                         []LinkItem    `json:"links"`
+	ComplianceProfileObjectGroups []ObjectGroup `json:"complianceProfileObjectGroups"`
```

Comment:
is this needed? I dont see it in the schema or swagger docs

Reply:
Ah good catch, I was attempting to fix an error and put that in but it ended u being silly. I'll go ahead and remove it.